pkgs/development/python-modules/spacy/default.nix

```diff
@@ -42,18 +45,21 @@ buildPythonPackage rec {
     jsonschema
     murmurhash
     numpy
+    packaging
+    pathy
     preshed
+    pydantic
     requests
     setuptools
-    srsly
     spacy-legacy
+    srsly
     thinc
-    wasabi
-    packaging
-    pathy
-    pydantic
+    tqdm
     typer
-  ] ++ lib.optional (pythonOlder "3.4") pathlib;
+    wasabi
+  ] ++ lib.optionals (pythonOlder "3.8") [
+    typing-extensions
+  ];
 
   checkInputs = [
     pytest
```

Comment:
Can we activate tests?

Reply:
And if we cannot activate tests then we might as well get rid of the commented out checkPhase and also remove checkInputs.